simtools/production_configuration/calculate_statistical_errors_grid_point.py

```diff
@@ -0,0 +1,458 @@
+"""
+Provides functionality to evaluate statistical errors from dl2_mc_events_file files.
```

Comment:
DL2 MC Event list files. Please change this throughout this file in docstrings.

Better would also to talk about uncertainties, not errors. But let's leave it.